kairon/shared/account/processor.py

```
@@ -789,6 +789,17 @@ def update_is_onboarded(email: Text):
         try:
             user = User.objects(email__iexact=email, status=True).get()
             user.is_onboarded = True
+            user.onboarding_timestamp = datetime.utcnow()
+            user.save()
+        except DoesNotExist as e:
+            logging.error(e)
+            raise AppException("User does not exists!")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)